utilities/speedrank-utils.js

```diff
@@ -0,0 +1,303 @@
+                continue;
+              }
+              else {
+                fightNames.push(fightRow[i]);
+              } 
+            }
+
+
+            let contestants = [];
+            let padLength = 0;
+
+            // Now that we know the fights, let's search each fight column for the player in question
+           for (let catName of fightNames) {
+              let categoryRange = find(catName, data.values);
+
+              //TODO: Get max rows of table
+              for (let i = 0; i < 500; i++) {
+                // categoryRange.row + 2 will give us the starting row of
+                // contestants.
+                let row = categoryRange.row + 2 + i;
+                //console.log("row = " + row);
+                // Stop processing if we've hit the end of the list.
+                if (data.values[row] === undefined) {
+                  break;
+                }
+                let contestant = [];
+                // entryStartPos gives us the starting cell
+                // with the contestant name.
+                const entryStartPos = categoryRange.columnNum - 1;
+                let cell = checkCell(data.values[row][entryStartPos]);
+                if (catName.length > padLength) {
+                  padLength = catName.length + 1;
+                  console.log(padLength);
+                }
+                if (cell.toLowerCase() === player.toLowerCase()) {
+                  contestant.push(catName);
+                  // Grab the actual rank of this row
+                  contestant.push(checkCell(data.values[row][0]));
+
+                  // The Overall category and Torment sheets places avg time in the third column. Others have them in the second
+                  if (catName === "Overall" || categorySheet === 'Torment'){
+                    contestant.push(checkCell(data.values[row][entryStartPos+2]));
+                  }
+                  else {
+                    contestant.push(checkCell(data.values[row][entryStartPos+1]));
+                  }
+
+                  contestants.push(contestant);
+                  break;
+                }
+              }
+            }
+            const rankTable =
+              outputRankTable(categorySheet, player,
+                categoryNames, contestants, padLength);
+            msg.channel.send(rankTable)
+              .then( (res) => {
+                resolve(res);
+            });
+          }
+        });
+      });
+  });
+};
+
+/** getSheet():
+ *  returns a sheet based on the shorthand input the user provides.
+ * @param {String} category: the category the user inputs.
+ * @param {String} secondaryCategory: the secondaryCategory the user inputs.
+ * @return {String} sheet
+ **/
+function getSheet(category, secondaryCategory) {
+  let version = "Overall";
+  if (secondaryCategory === 'no-csb'){
+    version = 'No CSB';
+  }
+  switch (category) {
+    case '3star':
+      category = 'GL 3* '+version+ ' rankings';
```

Comment:
For consistency's sake, I'd like for you to use Javascript's [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to match the rest of the documentation for anything here. For example, instead of:
```javascript
category = 'GL 3* '+version+ ' rankings';
```
Try:

```javascript
`category = GL 3* ${version} rankings;`
```